docker/dataset_eval/dataset_eval.service

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd /code
```

Comment:
Is this strictly needed ?
I mean:

```bash
#!/bin/bash
exec python /code/worker_manage.py hl_extractor
```

Or change to `/code` is required ?
In this case, better `cd /code && exec ....`

Reply:
again, this has been like this since we released. I'm happy to change to either of the two options, as long as it still works.

This will be running as root too, if you want us to run as unprivileged user we'll have to use a uid-dropping script. Does phusion provide one?

Reply:
It looks like the runit way of doing this is by using `exec chpst -ufoo:group python ....`

http://smarden.org/runit/faq.html#userservices